src/bin/lpython.cpp

```diff
@@ -1246,6 +1246,9 @@ int link_executable(const std::vector<std::string> &infiles,
         cmd += " -I " + rtlib_header_dir;
         cmd += " -L" + base_path
             + " -Wl,-rpath," + base_path + " -l" + runtime_lib + " -lm";
+        std::string py_version = "3.10";
+        std::string py_flags = R"(-I $CONDA_PREFIX/include/python)" + py_version + R"( -L$CONDA_PREFIX/lib -Wl,-rpath -Wl,$CONDA_PREFIX/lib -lpython)" + py_version + R"()";
+        cmd += " " + py_flags;
```

Comment:
I think this should be optional. We need these flags when there is a `@pythoncall()` in the user source code. Shall we add a compiler flag `--enable-cpython-runtime`? We also need the python version here so that we can link to the `pythonlib`. How do we get the python version that is installed on the system?

Reply:
Maybe just `--enable-cpython` and the Python version can be either specified at the command line. Alternatively we can specify the include path and the library name and path directly.

Reply:
I added the flag `--enable-cpython`. Currently, I hardcoded the python_version and the include paths for python library.